src/plugins/autoColumnSize/autoColumnSize.js

```diff
@@ -622,6 +624,15 @@ export class AutoColumnSize extends BasePlugin {
     privatePool.get(this).cachedColumnHeaders = this.hot.getColHeader();
   }
 
+  onAfterFormulasValuesUpdate(changes) {
+    const filteredChanges = arrayFilter(changes, change => change.address?.col !== undefined ?? false);
+    const changedColumns = arrayMap(filteredChanges, (change) => {
+      return this.hot.toPhysicalColumn(this.hot.propToCol(change.address.col));
+    });
```

Comment:
I don't think `change.address.col` coming from HF can ever be a `string`, so this line's basically translating a physical column to a visual column and then back to a physical column.

I think this should work just as fine.
```suggestion
    const changedColumns = arrayMap(filteredChanges, change => change.address.col);
```

Reply:
Done in eb94c0a